dev/benchmarks/macrobenchmarks/lib/src/shader_mask_cache.dart

```diff
@@ -43,6 +43,14 @@ class _ShaderMaskCachePageState extends State<ShaderMaskCachePage>
           buildShaderMask(0),
           const SizedBox(height: 10),
           buildShaderMask(1),
+          const SizedBox(height: 10),
+          buildShaderMaskWithBlendMode(BlendMode.modulate, 'BlendMode.modulate'),
+          const SizedBox(height: 10),
+          buildShaderMaskWithBlendMode(BlendMode.clear, 'BlendMode.clear'),
+          const SizedBox(height: 10),
+          buildShaderMaskWithBlendMode(BlendMode.dst, 'BlendMode.dst'),
+          const SizedBox(height: 10),
+          buildShaderMaskWithBlendMode(BlendMode.src, 'BlendMode.src'),
```

Comment:
Are all of these items visible on our default test device which is a Moto G4?

Reply:
I just fetched this PR and ran this on my G4 and the dst is only partially visible some of the time. The src example never appears.